src/Features/VisualBasic/Portable/UseNullPropagation/VisualBasicUseNullPropagationCodeFixProvider.vb

```diff
@@ -18,10 +18,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseNullPropagation
             InvocationExpressionSyntax,
             MemberAccessExpressionSyntax,
             ConditionalAccessExpressionSyntax,
-            InvocationExpressionSyntax)
+            InvocationExpressionSyntax,
+            InvocationExpressionSyntax,
+            ArgumentListSyntax)
 
         <ImportingConstructor>
         Public Sub New()
         End Sub
+
+        Protected Overrides Function ElementBindingExpression(argumentList As ArgumentListSyntax) As InvocationExpressionSyntax
+            Return SyntaxFactory.InvocationExpression(Nothing, argumentList)
```

Comment:
Why isn't this calling Syntaxfactory.ElementBindingExpression then? And then why do we need the overrides at all?

(This PR seems to expose a common way to create these, and then the feature gets a way to avoid use of that API...?)

Reply:
So the design here is that you express the high level idea at the generator level, it then produces the right low-level C#/VB syntax for that concept.  It turns out that syntax-level concept of an `element binding` for VB *is* an invocation expressoin.  i.e. when you want to index into a nullable array in VB, you just write:   `a?(1)`, while in C# you do `a?[1]`.  

So this allows feature to say "i need to index into a nullable array" and get the right syntax without having to know about VB.

Reply:
Should we split the two PRs then?

Reply:
Up to you.  Seems excessive to me.  If you want that, LMK :)

Reply:
Hmm, we probably should, or at least rebase the changes and break up the history. If we have to revert either the API change or the behavior change we don't want to accidentally revert the other one with it.

Reply:
Sounds good.  i will do that now.